src/app/components/SovGenerationNFTS/index.tsx

```diff
@@ -87,49 +92,62 @@ export function SovGenerationNFTS() {
-                className="md:tw-mr-5 sm:tw-mb-5 tw-mb-12 tw-ml-4 tw-mr-4 tw-relative tw-inline-block"
-              >
-                <div className="image-bordered">
-                  <img
-                    className="tw-relative tw-w-full tw-max-w-xs tw-h-full tw-rounded-md"
-                    src={item.image}
-                    alt=""
-                  />
-                </div>
-                <div className="sov-table">
-                  <table className="table">
-                    <tbody>
-                      <tr>
-                        <td>
-                          <b>SOV Genesis Pre-Order</b>
-                        </td>
-                        <td rowSpan={2}>
-                          <img src={sov_icon} alt="icon" />
-                        </td>
-                      </tr>
-                      <tr>
-                        <td>
-                          <b>Purchase Limit: {item.max} BTC</b>
-                        </td>
-                      </tr>
-                      <tr>
-                        <td>{item.title} Tier</td>
-                        <td>
-                          <div>SOV-OG</div>
-                        </td>
-                      </tr>
-                    </tbody>
-                  </table>
-                </div>
-              </div>
-            )
-          );
-        })}
+        {tiers.map((item, index) => (
+          <div
+            key={index}
+            className="md:tw-mr-5 sm:tw-mb-5 tw-mb-12 tw-ml-4 tw-mr-4 tw-relative tw-inline-block"
+          >
+            <div className="image-bordered">
+              <img
+                className="tw-relative tw-w-full tw-max-w-xs tw-h-full tw-rounded-md"
+                src={item.image}
+                alt=""
+              />
+            </div>
+            <div className="sov-table">
+              <table className="table">
+                <tbody>
+                  <tr>
+                    <td>
+                      <b>{t(translations.sovGenerationNFTS.genesisPreOrder)}</b>
+                    </td>
+                    <td rowSpan={2}>
+                      <img src={sov_icon} alt="icon" />
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <b>
+                        {t(translations.sovGenerationNFTS.purchaseLimit)}{' '}
+                        {item.max} BTC
+                      </b>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      {item.title} {t(translations.sovGenerationNFTS.tier)}
+                    </td>
+                    <td>
+                      <div>{t(translations.sovGenerationNFTS.tierSovOG)}</div>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+        ))}
+        {!items?.length && !tiers?.length && balanceBday === '0' && (
```

Comment:
Maybe also add `loading` check from `useLoadSovNfts` and `useCacheCallWithValue` (for community tiers). Currently it shows that user has no NFTs while it's being loaded